x/feeshare/ante/ante_test.go

```diff
@@ -0,0 +1,109 @@
+package ante_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+
+	ante "github.com/CosmosContracts/juno/v12/x/feeshare/ante"
+)
+
+type AnteTestSuite struct {
+	suite.Suite
+}
+
+func TestAnteSuite(t *testing.T) {
+	suite.Run(t, new(AnteTestSuite))
+}
+
+func (suite *AnteTestSuite) TestFeeLogic() {
+
+	// We expect all to pass
+	feeCoins := sdk.NewCoins(sdk.NewCoin("ujuno", sdk.NewInt(500)))
```

Comment:
Are there any tests for other denoms or fees payed in multiple tokens?

Reply:
We only accept juno as payout per the ante (tax reasons alex brought up monday)